components/in-the-news/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://localhost:3000/schemas/v1.json#",
   "name": "in-the-news",
-  "version": "2.0.0",
+  "version": "2.0.2",
```

Comment:
Just double checking, this got bumped up 2 versions I see. Assuming that's correct but adding a comment in case this needs to be 2.0.1

Reply:
The double bumps are because I had to make changes after my initial deploys to dev with further changes, redeploying to the Squiz Component Service requires a new version number each time.